airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/NormalizationActivityImpl.java

```diff
@@ -123,7 +129,8 @@ public NormalizationSummary normalize(final JobRunConfig jobRunConfig,
       final var fullDestinationConfig = secretsHydrator.hydrate(input.getDestinationConfiguration());
       final var fullInput = Jsons.clone(input).withDestinationConfiguration(fullDestinationConfig);
 
-      if (FeatureFlagHelper.isStrictComparisonNormalizationEnabledForWorkspace(featureFlags, input.getWorkspaceId())) {
+      if (FeatureFlagHelper.isStrictComparisonNormalizationEnabledForWorkspace(featureFlags, input.getWorkspaceId())
```

Comment:
Just curious - what are the two different scenarios that this `||` condition covers?

Reply:
`FeatureFlagHelper` is the legacy-style env var feature flags (hardcoded https://github.com/airbytehq/airbyte-platform-internal/blob/master/infra/gcp/env/prod/helm-data/platform-vars.yaml#L10)

`featureFlagClient` calls LD (using https://app.launchdarkly.com/default/production/features/normalization.strictComparisonNormalizationEnabled/targeting ).

so eventually I want to migrate everything to LD (or at least, any new flags should only use LD). Just didn't want to do that migration in the initial LD rollout.